core/server/helpers/body_class.js

```diff
@@ -2,48 +2,44 @@
 // Usage: `{{body_class}}`
 //
 // Output classes for the body element
-var proxy = require('./proxy'),
-    _ = require('lodash'),
-    SafeString = proxy.SafeString;
+const {SafeString} = require('./proxy');
 
-// We use the name body_class to match the helper for consistency:
+// We use the name body_class to match the helper for consistency
 module.exports = function body_class(options) { // eslint-disable-line camelcase
     let classes = [];
     const context = options.data.root.context;
     const obj = this.post || this.page;
     const tags = obj && obj.tags ? obj.tags : [];
     const isPage = !!(obj && obj.page);
 
-    if (_.includes(context, 'home')) {
+    if (context.includes('home')) {
         classes.push('home-template');
-    } else if (_.includes(context, 'post') && obj) {
+    } else if (context.includes('post') && obj) {
         classes.push('post-template');
-    } else if (_.includes(context, 'page') && obj && isPage) {
+    } else if (context.includes('page') && obj && isPage) {
         classes.push('page-template');
-        classes.push('page-' + obj.slug);
-    } else if (_.includes(context, 'tag') && this.tag) {
+        classes.push(`page-${obj.slug}`);
+    } else if (context.includes('tag') && this.tag) {
         classes.push('tag-template');
-        classes.push('tag-' + this.tag.slug);
-    } else if (_.includes(context, 'author') && this.author) {
+        classes.push(`tag-${this.tag.slug}`);
+    } else if (context.includes('author') && this.author) {
         classes.push('author-template');
-        classes.push('author-' + this.author.slug);
-    } else if (_.includes(context, 'private')) {
+        classes.push(`author-${this.author.slug}`);
+    } else if (context.includes('private')) {
         classes.push('private-template');
     }
 
     if (tags) {
-        classes = classes.concat(tags.map(function (tag) {
-            return 'tag-' + tag.slug;
-        }));
+        classes = classes.concat(
+            tags.map(({slug}) => `tag-${slug}`)
+        );
     }
 
-    if (_.includes(context, 'paged')) {
+    if (context.includes('paged')) {
         classes.push('paged');
     }
 
-    classes = _.reduce(classes, function (memo, item) {
-        return memo + ' ' + item;
-    }, '');
+    classes = classes.reduce((memo, item) => `${memo} ${item}`, '');
```

Comment:
I'm not entirely sure why we don't do `classes.join(' ').trimm()` 🤔

Reply:
No reason not to, and IMO `join` is preferred as it is less cognitive overhead to understand